Formula/recipes.rb

```diff
@@ -0,0 +1,24 @@
+class Recipes < Formula
+  desc "Formula for GNOME recipes"
+  homepage "https://wiki.gnome.org/Apps/Recipes"
+  url "https://mclasen.fedorapeople.org/recipes-0.4.2.tar.xz"
```

Comment:
This URL is failing: https://bot.brew.sh/job/Homebrew%20Core/13831/version=sierra/testReport/junit/brew-test-bot/sierra/fetch_recipes/